lib/tests/helpers.rs

```diff
@@ -255,6 +254,19 @@ impl Test {
 		})
 	}
 
+	#[allow(dead_code)]
```

Comment:
```suggestion
```

I think it's not actually dead code because we do use it in the `lib/tests/vector.rs` test.